app/views/setup/welcome.html.slim

```diff
@@ -156,5 +156,20 @@ h1 Initial CaaS Platform Configuration
   = render partial: 'setup/cloud/settings', locals: { f: f }
   = render partial: 'setup/settings/container_runtime', locals: { f: f }
 
+  .panel.panel-default
+    .panel-heading.clearfix
+      h3.panel-title
+        | Certificate
```

Comment:
I was wondering.... Maybe it would be a good idea to rename this to "System wide certificate". Or at least be more explicit on the description of this config?